libr/core/cmd_api.c

```diff
@@ -1162,6 +1235,19 @@ R_API RCmdDesc *r_cmd_desc_parent(RCmdDesc *cd) {
 	return cd->parent;
 }
 
+R_API bool r_cmd_desc_has_handler(RCmdDesc *cd) {
```

Comment:
this function only can return true if the .cb is set on argv mode, all the other cases is false, can you simplify this logic to just handle this?

Reply:
No there is also the old input case that can return true